cmd/utils/flags.go

```diff
@@ -385,6 +386,21 @@ var (
 		Usage: "HTTP-RPC server listening port",
 		Value: node.DefaultHTTPPort,
 	}
+	RPCReadTimeoutFlag = cli.DurationFlag{
+		Name:  "rpcreadtimeout",
+		Usage: "HTTP-RPC server read timeout",
+		Value: rpc.DefaultHTTPReadTimeout,
+	}
+	RPCWriteTimeoutFlag = cli.DurationFlag{
+		Name:  "rpcwritetimeout",
+		Usage: "HTTP-RPC server write timeout",
+		Value: rpc.DefaultHTTPWriteTimeout,
+	}
+	RPCIdleTimeoutFlag = cli.DurationFlag{
+		Name:  "rpcidletimeout",
+		Usage: "HTTP-RPC server idle timeout",
+		Value: rpc.DefaultHTTPIdleTimeout,
+	}
```

Comment:
IMHO flags are not needed for this. Very few people will set this option and they can use the config file.

Reply:
@fjl the majority of configs can be set from command line so `config.toml` method is confusing a bit. It would be great to be able to configure this using command line flags.